### PR TITLE
refactor(db): optimize logs for db module

### PR DESCRIPTION
**Pull request by halibobo1205**
1. update log levels
  2. optimize log contents
  3. add some necessary logs

**What does this PR do?**

**Why are these changes required?**

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: chainbase/src/main/java/org/tron/core/ChainBaseManager.java [hunk 1]
```diff
@@ -233,13 +233,13 @@ public class ChainBaseManager {
   private DbStatService dbStatService;
 
   public void closeOneStore(ITronChainBase database) {
-    logger.info("******** begin to close " + database.getName() + " ********");
+    logger.info("******** Begin to close {}  ********",  database.getName());
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/ChainBaseManager.java:236]: two blank space

File: chainbase/src/main/java/org/tron/core/db/KhaosDatabase.java [hunk 2]
```diff
@@ -110,12 +110,13 @@ public BlockCapsule push(BlockCapsule blk)
       if (kblock != null) {
         if (blk.getNum() != kblock.num + 1) {
           throw new BadNumberBlockException(
-              "parent number :" + kblock.num + ",block number :" + blk.getNum());
+              String.format("parent number :%d , block number :%d", kblock.num, blk.getNum())
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db/KhaosDatabase.java:113]: number: %d

File: chainbase/src/main/java/org/tron/core/db/KhaosDatabase.java [hunk 3]
```diff
@@ -110,12 +110,13 @@ public BlockCapsule push(BlockCapsule blk)
...
+              );
         }
         block.setParent(kblock);
       } else {
         miniUnlinkedStore.insert(block);
-        logger.error("blk:{}, head:{}, miniStore:{}, miniUnlinkedStore:{}",
+        logger.error("Block:{}, head:{}, miniStore:{}, miniUnlinkedStore:{}",
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db/KhaosDatabase.java:119]: : {}

File: chainbase/src/main/java/org/tron/core/db/StorageMarket.java [hunk 4]
```diff
@@ -23,12 +20,12 @@ public StorageMarket(AccountStore accountStore, DynamicPropertiesStore dynamicPr
   }
 
   private long exchangeToSupply(boolean isTRX, long quant) {
-    logger.info("isTRX: " + isTRX);
+    logger.info("IsTRX: {} ", isTRX);
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db/StorageMarket.java:23]: {}"

File: chainbase/src/main/java/org/tron/core/db/StorageMarket.java [hunk 5]
```diff
@@ -95,45 +92,45 @@ public long calculateTax(long duration, long limit) {
...
     long newTotalReserved = dynamicPropertiesStore.getTotalStorageReserved()
         + storageTax;
-    logger.info("reserved: " + dynamicPropertiesStore.getTotalStorageReserved());
+    logger.info("Reserved: {}", dynamicPropertiesStore.getTotalStorageReserved());
     boolean eq = dynamicPropertiesStore.getTotalStorageReserved()
         == 128L * 1024 * 1024 * 1024;
-    logger.info("reserved == 128GB: " + eq);
-    logger.info("newTotalTax: " + newTotalTax + "  newTotalPool: " + newTotalPool
-        + NEW_TOTAL_RESERVED + newTotalReserved);
+    logger.info("Reserved == 128GB: {} ", eq);
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db/StorageMarket.java:112]: {}"

File: chainbase/src/main/java/org/tron/core/db/TransactionTrace.java [hunk 6]
```diff
@@ -269,14 +268,15 @@ public void check() throws ReceiptCheckErrException {
...
+      throw new ReceiptCheckErrException(
+          String.format("null resultCode id: %s",
+              Hex.toHexString(trx.getTransactionId().getBytes())));
     }
     if (!trx.getContractRet().equals(receipt.getResult())) {
-      logger.info(
-          "this tx id: {}, the resultCode in received block: {}, the resultCode in self: {}",
+      throw new ReceiptCheckErrException(String.format(
+          "Different resultCode id: %s, expect: %s, actual: %s",
           Hex.toHexString(trx.getTransactionId().getBytes()), trx.getContractRet(),
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db/TransactionTrace.java:278]: Hex.toHexString(trx.getTransactionId().getBytes())   --->  trx.getTransactionId()

File: chainbase/src/main/java/org/tron/core/db2/common/TxCacheDB.java [hunk 7]
```diff
@@ -109,8 +109,7 @@ private void initCache() {
       bloomFilters[1].put(entry.getKey());
       persistentSize++;
     }
-    logger.info("load transaction cache from persistentStore "
-            + "db-size:{}, filter-size:{}, filter-fpp:{}, cost:{}ms",
+    logger.info("Load cache from persistentStore : db:{}, filter:{}, filter-fpp:{}, cost:{} ms",
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db2/common/TxCacheDB.java:112]: : {}

File: chainbase/src/main/java/org/tron/core/db2/common/TxCacheDB.java [hunk 8]
```diff
@@ -133,8 +132,7 @@ private void init() {
       trx.getTransactionIds().forEach(tid -> bloomFilters[1].put(Hex.decode(tid)));
     }
 
-    logger.info("load transaction cache from recentTransactionStore"
-            + " filter-size:{}, filter-fpp:{}, cost:{}ms",
+    logger.info("Load cache from recentTransactionStore : filter:{}, filter-fpp:{}, cost:{} ms",
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db2/common/TxCacheDB.java:135]: recentTransactionStore, filter: {}, : {}, : {}

File: chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java [hunk 9]
```diff
@@ -150,7 +150,7 @@ private void retreat() {
 
   public void merge() {
     if (activeSession <= 0) {
-      throw new RevokingStoreIllegalStateException("activeDialog has to be greater than 0");
+      throw new RevokingStoreIllegalStateException(activeSession);
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java:153]: why delete the error content?

File: chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java [hunk 10]
```diff
@@ -312,7 +314,7 @@ public void flush() {
         long checkPointEnd = System.currentTimeMillis();
         refresh();
         flushCount = 0;
-        logger.info("flush cost:{}, create checkpoint cost:{}, refresh cost:{}",
+        logger.info("Flush cost:{}, create checkpoint cost:{}, refresh cost:{}",
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java:317]: : {}

File: chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java [hunk 11]
```diff
@@ -480,17 +482,7 @@ public void destroy() {
...
-          snapshotManager.revoke();
-        }
-      } catch (Exception e) {
-        logger.error("revoke database error.", e);
-        throw new RevokingStoreIllegalStateException(e);
-      }
-      if (disableOnExit) {
-        snapshotManager.disable();
-      }
+     destroy();
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java:485]: should not change the logic code when just formatting the log

File: chainbase/src/main/java/org/tron/core/service/MortgageService.java [hunk 12]
```diff
@@ -125,7 +125,7 @@ public void withdrawReward(byte[] address) {
         reward = computeReward(beginCycle, endCycle, account);
         adjustAllowance(address, reward);
         reward = 0;
-        logger.info("latest cycle reward {},{}", beginCycle, account.getVotesList());
+        logger.info("Latest cycle reward {},{}", beginCycle, account.getVotesList());
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/service/MortgageService.java:128]: {}, {}

File: chainbase/src/main/java/org/tron/core/service/MortgageService.java [hunk 13]
```diff
@@ -252,7 +252,7 @@ public void adjustAllowance(byte[] address, long amount) {
       }
       adjustAllowance(accountStore, address, amount);
     } catch (BalanceInsufficientException e) {
-      logger.error("withdrawReward error: {},{}", Hex.toHexString(address), address, e);
+      logger.error("WithdrawReward error: {}", e.getMessage());
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/service/MortgageService.java:255]: why delete the addr info?

File: chainbase/src/main/java/org/tron/core/service/MortgageService.java [hunk 14]
```diff
@@ -194,7 +194,7 @@ private long computeReward(long cycle, AccountCapsule accountCapsule) {
       long userVote = vote.getVoteCount();
       double voteRate = (double) userVote / totalVote;
       reward += voteRate * totalReward;
-      logger.debug("computeReward {} {} {} {},{},{},{}", cycle,
+      logger.debug("ComputeReward {} {} {} {},{},{},{}", cycle,
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/service/MortgageService.java:197]: why only three ","?

File: chainbase/src/main/java/org/tron/core/service/MortgageService.java [hunk 15]
```diff
@@ -266,7 +266,8 @@ public void adjustAllowance(AccountStore accountStore, byte[] accountAddress, lo
 
     if (amount < 0 && allowance < -amount) {
       throw new BalanceInsufficientException(
-          StringUtil.createReadableString(accountAddress) + " insufficient balance");
+          String.format("%s insufficient balance,amount: %d,allowance: %d",
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/service/MortgageService.java:269]: , amount
, allowance

File: chainbase/src/main/java/org/tron/core/store/DynamicPropertiesStore.java [hunk 16]
```diff
@@ -905,7 +905,7 @@ public long getAbiMoveDone() {
   }
 
   public void saveBlockFilledSlotsIndex(int blockFilledSlotsIndex) {
-    logger.debug("blockFilledSlotsIndex:" + blockFilledSlotsIndex);
+    logger.debug("BlockFilledSlotsIndex:" + blockFilledSlotsIndex);
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/store/DynamicPropertiesStore.java:908]: : {}"

File: chainbase/src/main/java/org/tron/core/store/TreeBlockIndexStore.java [hunk 17]
```diff
@@ -28,7 +28,7 @@ public byte[] get(Long num)
       throws ItemNotFoundException {
     BytesCapsule value = getUnchecked(ByteArray.fromLong(num));
     if (value == null || value.getData() == null) {
-      throw new ItemNotFoundException("number: " + num + " is not found!");
+      throw new ItemNotFoundException(String.format("number:%d is not found!", num));
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/store/TreeBlockIndexStore.java:31]: : %d

File: chainbase/src/main/java/org/tron/core/store/TreeBlockIndexStore.java [hunk 18]
```diff
@@ -39,7 +39,8 @@ public BytesCapsule get(byte[] key)
       throws ItemNotFoundException {
     byte[] value = revokingDB.getUnchecked(key);
     if (ArrayUtils.isEmpty(value)) {
-      throw new ItemNotFoundException("number: " + Arrays.toString(key) + " is not found!");
+      throw new ItemNotFoundException(String.format("number:%d is not found!",
```
Comment (guoquanwu) [inline chainbase/src/main/java/org/tron/core/store/TreeBlockIndexStore.java:42]: : %d

File: framework/src/main/java/org/tron/core/db/BandwidthProcessor.java [hunk 19]
```diff
@@ -112,7 +113,7 @@ public void consume(TransactionCapsule trx, TransactionTrace trace)
         bytesSize += Constant.MAX_RESULT_SIZE_IN_TX;
       }
 
-      logger.debug("trxId {}, bandwidth cost: {}", trx.getTransactionId(), bytesSize);
+      logger.debug("TrxId {}, bandwidth cost: {}", trx.getTransactionId(), bytesSize);
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/core/db/BandwidthProcessor.java:116]: TxId

File: framework/src/main/java/org/tron/core/db/BandwidthProcessor.java [hunk 20]
```diff
@@ -144,8 +145,9 @@ public void consume(TransactionCapsule trx, TransactionTrace trace)
 
       long fee = chainBaseManager.getDynamicPropertiesStore().getTransactionFee() * bytesSize;
       throw new AccountResourceInsufficientException(
-          "Account has insufficient bandwidth[" + bytesSize + "] and balance["
-              + fee + "] to create new account");
+          String.format(
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/core/db/BandwidthProcessor.java:148]: Account %s, or change Account to account

File: framework/src/main/java/org/tron/core/db/BandwidthProcessor.java [hunk 21]
```diff
@@ -393,7 +395,8 @@ private boolean useAccountNet(AccountCapsule accountCapsule, long bytes, long no
     long newNetUsage = increase(netUsage, 0, latestConsumeTime, now);
 
     if (bytes > (netLimit - newNetUsage)) {
-      logger.debug("net usage is running out, now use free net usage");
+      logger.debug("{} bytes: {},netLimit:{},newNetUsage:{}",
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/core/db/BandwidthProcessor.java:398]: {}, netLimit: {}, newNetUsage: {}

File: framework/src/main/java/org/tron/core/db/BandwidthProcessor.java [hunk 22]
```diff
@@ -416,7 +419,8 @@ private boolean useFreeNet(AccountCapsule accountCapsule, long bytes, long now)
     long newFreeNetUsage = increase(freeNetUsage, 0, latestConsumeFreeTime, now);
 
     if (bytes > (freeNetLimit - newFreeNetUsage)) {
-      logger.debug("free net usage is running out");
+      logger.debug("{} bytes: {},freeNetLimit:{},newFreeNetUsage:{}",
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/core/db/BandwidthProcessor.java:422]: {}, freeNetLimit: {}, newFreeNetUsage: {}

File: framework/src/main/java/org/tron/core/db/BandwidthProcessor.java [hunk 23]
```diff
@@ -427,7 +431,8 @@ private boolean useFreeNet(AccountCapsule accountCapsule, long bytes, long now)
     long newPublicNetUsage = increase(publicNetUsage, 0, publicNetTime, now);
 
     if (bytes > (publicNetLimit - newPublicNetUsage)) {
-      logger.debug("free public net usage is running out");
+      logger.debug("{} bytes: {},publicNetLimit:{},newPublicNetUsage:{}",
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/core/db/BandwidthProcessor.java:434]: same with above

File: framework/src/main/java/org/tron/core/db/PendingManager.java [hunk 24]
```diff
@@ -34,7 +34,7 @@ public void close() {
         Metrics.counterInc(MetricKeys.Counter.TXS, 1,
             MetricLabels.Counter.TXS_FAIL, MetricLabels.Counter.TXS_FAIL_TIMEOUT);
         if (Args.getInstance().isOpenPrintLog()) {
-          logger.warn("[timeout] remove tx from repush, txId:{}", tx.getTransactionId());
+          logger.warn("Timeout remove tx from repush, txId:{}", tx.getTransactionId());
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/core/db/PendingManager.java:37]: : {}

File: framework/src/main/java/org/tron/core/db/PendingManager.java [hunk 25]
```diff
@@ -54,7 +54,7 @@ public void close() {
     Metrics.gaugeSet(MetricKeys.Gauge.MANAGER_QUEUE, 0,
         MetricLabels.Gauge.QUEUE_POPPED);
     if (Args.getInstance().isOpenPrintLog()) {
-      logger.warn("pending tx size:{}", dbManager.getRePushTransactions().size());
+      logger.info("Pending tx size:{}", dbManager.getRePushTransactions().size());
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/core/db/PendingManager.java:57]: size: {}

File: framework/src/main/java/org/tron/core/db/PendingManager.java [hunk 26]
```diff
@@ -69,7 +69,7 @@ private void txIteration(TransactionCapsule tx) {
         Metrics.counterInc(MetricKeys.Counter.TXS, 1,
             MetricLabels.Counter.TXS_FAIL, MetricLabels.Counter.TXS_FAIL_TIMEOUT);
         if (Args.getInstance().isOpenPrintLog()) {
-          logger.warn("[timeout] remove tx from pending, txId:{}", tx.getTransactionId());
+          logger.warn("Timeout remove tx from pending, txId:{}", tx.getTransactionId());
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/core/db/PendingManager.java:72]: txId: {}

File: framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java [hunk 27]
```diff
@@ -86,11 +86,11 @@ public void generateSnapshot(String sourceDir, String snapshotDir) {
       fillSnapshotBlockAndTransDb(sourceDir, snapshotDir);
       generateInfoProperties(Paths.get(snapshotDir, INFO_FILE_NAME).toString(), sourceDir);
     } catch (IOException | RocksDBException e) {
-      logger.error("create snapshot failed, " + e.getMessage());
+      logger.error("Create snapshot failed,{} ", e.getMessage());
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java:89]: , {}"

File: framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java [hunk 28]
```diff
@@ -431,14 +433,14 @@ private static boolean isEmptyBytes(byte[] b) {
...
-    logger.info("-- delete the info file.");
+    logger.info("Delete the info file from {}.", databaseDir);
     Files.delete(Paths.get(databaseDir, INFO_FILE_NAME));
     if (!isLite(databaseDir)) {
       DBInterface destCommonDb = DbTool.getDB(databaseDir, COMMON_DB_NAME);
       destCommonDb.delete(DB_KEY_NODE_TYPE);
       destCommonDb.delete(DB_KEY_LOWEST_BLOCK_NUM);
-      logger.info("-- deleted node_type and lowest_block_num  from  "
-          + "common to identify this node is a real fullnode.");
+      logger.info("Deleted {}  and {}   from {}  to identify this node is a real fullnode.",
```
Comment (guoquanwu) [inline framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java:442]: Deleted {} and {} from {} 

too many blank spaces

**Review by guoquanwu — APPROVED** [review]

File: framework/src/main/java/org/tron/core/db/Manager.java [hunk 29]
```diff
@@ -2191,23 +2167,23 @@ private void postTransactionTrigger(final TransactionCapsule trxCap,
...
+      logger.info("Switch fork occurred, post reOrgContractTrigger.");
       try {
         BlockCapsule oldHeadBlock = chainBaseManager.getBlockById(
             getDynamicPropertiesStore().getLatestBlockHeaderHash());
         for (TransactionCapsule trx : oldHeadBlock.getTransactions()) {
           postContractTrigger(trx.getTrxTrace(), true, oldHeadBlock.getBlockId().toString());
         }
       } catch (BadItemException | ItemNotFoundException e) {
-        logger.error("block header hash does not exist or is bad: {}",
+        logger.error("Block header hash does not exist or is bad: {}.",
```
Comment (lurais) [inline framework/src/main/java/org/tron/core/db/Manager.java:2186]: Shall we print the error message detail?

File: framework/src/main/java/org/tron/core/db/accountstate/callback/AccountStateCallBack.java [hunk 30]
```diff
@@ -84,10 +84,10 @@ public void executePushFinish() throws BadBlockException {
       newRoot = Hash.EMPTY_TRIE_HASH;
     }
     if (!oldRoot.isEmpty() && !Arrays.equals(oldRoot.toByteArray(), newRoot)) {
-      logger.error("the accountStateRoot hash is error. {}, oldRoot: {}, newRoot: {}",
-          blockCapsule, ByteArray.toHexString(oldRoot.toByteArray()),
-          ByteArray.toHexString(newRoot));
-      throw new BadBlockException("the accountStateRoot hash is error");
+      throw new BadBlockException(
+          String.format("accountStateRoot hash is error, %s, oldRoot: %s, newRoot: %s.",
```
Comment (lurais) [inline framework/src/main/java/org/tron/core/db/accountstate/callback/AccountStateCallBack.java:88]: Should the begin character of the log be  a capital ?

File: framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java [hunk 31]
```diff
@@ -95,16 +95,16 @@ public void doBackup(BlockCapsule block) {
           deleteBackup(DB_BACKUP_INDEX2);
           break;
         default:
-          logger.warn("invalid backup state");
+          logger.warn("invalid backup state {}.", getBackupState());
```
Comment (lurais) [inline framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java:98]: Should the begin character of the log be  a capital ?

File: framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java [hunk 32]
```diff
@@ -115,7 +115,7 @@ private void backup(int i) throws RocksDBException {
     } else if (i == DB_BACKUP_INDEX2) {
       path = parameter.getDbBackupConfig().getBak2path();
     } else {
-      throw new RuntimeException("Error backup with undefined index");
+      throw new RuntimeException(String.format("error backup with undefined index %d", i));
```
Comment (lurais) [inline framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java:118]: Should the begin character of the log be  a capital ?

File: framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java [hunk 33]
```diff
@@ -134,7 +134,7 @@ private void deleteBackup(int i) {
     } else if (i == DB_BACKUP_INDEX2) {
       path = parameter.getDbBackupConfig().getBak2path();
     } else {
-      throw new RuntimeException("Error deleteBackup with undefined index");
+      throw new RuntimeException(String.format("error deleteBackup with undefined index %d", i));
```
Comment (lurais) [inline framework/src/main/java/org/tron/core/db/backup/BackupDbUtil.java:137]: Should the begin character of the log be  a capital ?

File: framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java [hunk 34]
```diff
@@ -180,22 +180,22 @@ private void mergeCheckpoint2History(String sourceDir, String destDir) {
   }
 
   private void split(String sourceDir, String destDir, List<String> dbs) throws IOException {
-    logger.info("-- begin to split the dbs.");
+    logger.info("Begin to split the dbs.");
     if (!new File(sourceDir).isDirectory()) {
-      throw new RuntimeException("sourceDir must be a directory, sourceDir: " + sourceDir);
+      throw new RuntimeException("SourceDir must be a directory, sourceDir: " + sourceDir);
```
Comment (lurais) [inline framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java:185]: Use {} and "," to replace the operator "+"?

File: framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java [hunk 35]
```diff
@@ -180,22 +180,22 @@ private void mergeCheckpoint2History(String sourceDir, String destDir) {
...
     }
     File destPath = new File(destDir);
     if (new File(destDir).exists()) {
-      throw new RuntimeException("destDir is already exist, please remove it first");
+      throw new RuntimeException("DestDir is already exist, please remove it first");
     }
     if (!destPath.mkdirs()) {
-      throw new RuntimeException("destDir create failed, please check");
+      throw new RuntimeException("DestDir create failed, please check");
```
Comment (lurais) [inline framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java:192]: Shall we print the destDir?

File: framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java [hunk 36]
```diff
@@ -331,38 +331,40 @@ private BlockNumInfo checkAndGetBlockNumInfo(String historyDir, String databaseD
...
   }
 
   private void backupArchiveDbs(String databaseDir) throws IOException {
     String bakDir = String.format("%s%s%s%d",
             databaseDir, File.separator, BACKUP_DIR_PREFIX, START_TIME);
-    logger.info("-- backup the archive dbs to {}", bakDir);
+    logger.info("Backup the archive dbs to {}.", bakDir);
     if (!FileUtil.createDirIfNotExists(bakDir)) {
-      throw new RuntimeException("create bak dir failed");
+      throw new RuntimeException(String.format("create bak dir %s failed", bakDir));
```
Comment (lurais) [inline framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java:354]: Should the begin character of the log be  a capital ?

File: framework/src/test/java/org/tron/common/storage/leveldb/RocksDbDataSourceImplTest.java [hunk 37]
```diff
@@ -271,7 +271,9 @@ public void testCheckOrInitEngine() {
     try {
       dataSource.initDB();
     } catch (Exception e) {
-      Assert.assertTrue(e.getMessage().contains("Failed to"));
+      Assert.assertEquals(String.format("failed to check database: %s, engine do not match",
```
Comment (lurais) [inline framework/src/test/java/org/tron/common/storage/leveldb/RocksDbDataSourceImplTest.java:274]: Should the begin character of the log be  a capital ?

File: framework/src/test/java/org/tron/core/db/ManagerTest.java [hunk 38]
```diff
@@ -298,7 +298,8 @@ public void adjustBalanceTest() {
       Assert.assertTrue(false);
     } catch (BalanceInsufficientException e) {
       Assert.assertEquals(
-          StringUtil.createReadableString(account.createDbKey()) + " insufficient balance",
+          StringUtil.createReadableString(account.createDbKey()) + " insufficient balance"
```
Comment (lurais) [inline framework/src/test/java/org/tron/core/db/ManagerTest.java:301]: Use the {} and "," to replace the operator "+"?

File: framework/src/test/java/org/tron/core/db/ManagerTest.java [hunk 39]
```diff
@@ -354,7 +355,8 @@ public void adjustAssetBalanceV2Test() {
     } catch (BalanceInsufficientException e) {
       Assert.assertTrue(e instanceof BalanceInsufficientException);
       Assert.assertEquals(
-          "reduceAssetAmount failed !", e.getMessage());
+          "reduceAssetAmount failed! account: " + StringUtil.encode58Check(account.createDbKey()),
```
Comment (lurais) [inline framework/src/test/java/org/tron/core/db/ManagerTest.java:358]: Use the {} and "," to replace the operator "+"?

File: framework/src/test/java/org/tron/core/db/ManagerTest.java [hunk 40]
```diff
@@ -389,7 +391,8 @@ public void pushBlockInvalidMerkelRoot() {
       Assert.assertTrue(false);
     } catch (BadBlockException e) {
       Assert.assertTrue(e instanceof BadBlockException);
-      Assert.assertEquals("The merkle hash is not validated", e.getMessage());
+      Assert.assertEquals("The merkle hash is not validated for "
```
Comment (lurais) [inline framework/src/test/java/org/tron/core/db/ManagerTest.java:394]: Use the {} and "," to replace the operator "+"?

File: framework/src/test/java/org/tron/core/db/ManagerTest.java [hunk 41]
```diff
@@ -444,7 +447,8 @@ public void pushBlockTooMuchShieldedTransactions() {
       Assert.assertTrue(false);
     } catch (BadBlockException e) {
       Assert.assertTrue(e instanceof BadBlockException);
-      Assert.assertEquals("shielded transaction count > " + SHIELDED_TRANS_IN_BLOCK_COUNTS,
+      Assert.assertEquals("num: " + blockCapsule2.getNum()
```
Comment (lurais) [inline framework/src/test/java/org/tron/core/db/ManagerTest.java:450]: Use the {} and "," to replace the operator "+"?

**Review by lurais — COMMENTED** [review]
some comments added.

**Pull request merged**